source/slang/slang-syntax.h

```diff
@@ -549,23 +572,18 @@ namespace Slang
         return SyntaxClass<T>::getClass();
     }
 
-    NodeBase* _dynamicCastImpl(NodeBase* node, SyntaxClassBase const& toClass);
-
```

Comment:
Did you measure that moving the casting logic out of this subroutine and into the inline functions actually paid off? Usually I've been trying to put as *little* code into templated functions as possible, and always have them immediately delegate to a non-template function in order to add as little code size increase as possible.

Reply:
The inlining like this originally showed a small benefit. With the range change - the difference seems less obvious. With it being inlined and the amount of code being pretty small I'd expect it to be ok - but in might produce enough more code to impact code size perhaps.